src/Goerli-DssSpell.sol

```diff
@@ -19,13 +19,23 @@ pragma solidity 0.6.12;
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
 
+interface DssVestLike {
```

Comment:
I think we don't need the DssVestLike interface as it's already included in [dss-interfaces](https://github.com/makerdao/dss-interfaces/blob/master/src/dss/VestAbstract.sol)

Reply:
That interface is out of date. `init` is now `create`, `usr()` is also missing. Just going to stick with the inline interfaces for now.

Reply:
Oh! Good catch! 
We should fix that.